frontend/src/components/browse-sightings-page/MapPage.tsx

```diff
@@ -0,0 +1,43 @@
+import { Sighting } from "../../clients/apiClient";
+// import { SlippyMap, Marker, Label, InfoBox } from "react-slippy-map";
+import GoogleMapReact from "google-map-react";
+import { Marker } from "./marker";
+import React from "react";
+import { map } from "leaflet";
```

Comment:
I don't think you're using this? Also, it doesn't look like you've installed leaflet, so I'm surprised this runs at all? If you have installed it and aren't using it, please uninstall it